src/main/java/edu/wpi/first/shuffleboard/components/DashboardTabPane.java

```diff
@@ -60,8 +66,8 @@ public void addWidgetToActivePane(Widget widget) {
    */
   public void selectWidgets(Predicate<Widget> selector) {
     getTabs().stream().map(optionalCast(DashboardTab.class))
-            .forEach(tab -> tab.map(DashboardTab::getWidgetPane)
-                    .ifPresent(pane -> pane.selectWidgets(selector)));
+        .forEach(tab -> tab.map(DashboardTab::getWidgetPane)
+            .ifPresent(pane -> pane.selectWidgets(selector)));
```

Comment:
You should consider at least using an editor config file to make this consistent:
http://editorconfig.org/